test/Mono.Linker.Tests.Cases/DataFlow/MakeGenericDataFlow.cs

```diff
@@ -684,6 +686,12 @@ static void TestWithUnmanagedConstraint ()
 			{
 				var t = new T ();
 			}
+
+			static void TestGetMethodFromHandle (Type unknownType = null)
+			{
+				MethodInfo m = (MethodInfo) MethodInfo.GetMethodFromHandle (typeof (MakeGenericMethod).GetMethod (nameof (GenericWithNoRequirements)).MethodHandle);
+				m.MakeGenericMethod (unknownType);
```

Comment:
Just making sure I understand how the test works - is the idea that if we didn't support GetMethodFromHandle, the call to `MakeGenericMethod` would warn about an unknown method?

Might be worth adding a test that produces some warnings (where the generic parameters have requirements).

Reply:
Yes - if the method is not know MakeGenericMethod should always warn... so the fact that it doesn't warn means we were able to correctly flow the MethodInfo.
I'll add a negative test as well.